src/github.com/getlantern/flashlight/pac.go

```diff
@@ -41,16 +44,33 @@ func setUpPacTool() error {
 	return nil
 }
 
+func setProxyAddr(addr string) {
```

Comment:
Wait so what impact does this have on unblocked sites? The whole point of the PAC file is to make sure we don't slow down unblocked sites. Even if we go directly from Lantern, how much impact does that have versus going directly from the browser without Lantern for unblocked sites?